definitions/marts/bigquery_marts/ecf2/ecf2_teacher_induction_periods.sqlx

```diff
@@ -73,3 +118,11 @@ LEFT JOIN
   ${ref(`appropriate_bodies_latest_ecf2`)} AS abs
 ON
   induction_periods.appropriate_body_id = abs.id
+LEFT JOIN
+  induction_origins

```

Comment:
Personally I'd call these inductions_with_origin and inductions_with_change_counts or something so it's clear that one row is still an induction, not a change (or an origin)